samples/Ark.ReferenceProject/Core/Ark.Reference.Core.Tests/Features/BookPrintProcessSteps.cs

```diff
@@ -0,0 +1,214 @@
+        public void GivenIHaveCreatedABookWithTitleAndAuthor(string title, string author)
+        {
+            var request = new Book.V1.Create
+            {
+                Title = title,
+                Author = author,
+                Genre = BookGenre.Fiction,
+                ISBN = "978-0-123456-78-9"
+            };
+
+            _client.PostAsJson("book", request);
+            _client.ThenTheRequestSucceded();
+            
+            var result = _client.ReadAs<Book.V1.Output>();
+            _currentBookId = result.Id;
+        }
+
+        [Given(@"I have created a book print process for that book")]
+        [When(@"I create a book print process for that book")]
+        public void WhenICreateABookPrintProcessForThatBook()
+        {
+            var request = new BookPrintProcess.V1.Create
+            {
+                BookId = _currentBookId!.Value,
+                ShouldFail = false
+            };
+
+            _client.PostAsJson(_controllerName, request);
+                
+            if (_client.LastStatusCodeIsSuccess())
+            {
+                _currentPrintProcess = _client.ReadAs<BookPrintProcess.V1.Output>();
+                _currentPrintProcessId = _currentPrintProcess!.BookPrintProcessId;
+            }
+            else
+            {
+                _lastErrorContent = _client.ReadAsString();
+            }
+        }
+
+        [Given(@"I have created a book print process for that book with ShouldFail (.*)")]
+        public void GivenIHaveCreatedABookPrintProcessWithShouldFail(bool shouldFail)
+        {
+            var request = new BookPrintProcess.V1.Create
+            {
+                BookId = _currentBookId!.Value,
+                ShouldFail = shouldFail
+            };
+
+            _client.PostAsJson(_controllerName, request);
+            _client.ThenTheRequestSucceded();
+                
+            _currentPrintProcess = _client.ReadAs<BookPrintProcess.V1.Output>();
+            _currentPrintProcessId = _currentPrintProcess!.BookPrintProcessId;
+        }
+
+        [When(@"I try to create another book print process for that book")]
+        public void WhenITryToCreateAnotherBookPrintProcessForThatBook()
+        {
+            var request = new BookPrintProcess.V1.Create
+            {
+                BookId = _currentBookId!.Value,
+                ShouldFail = false
+            };
+
+            _client.PostAsJson(_controllerName, request);
+                
+            if (!_client.LastStatusCodeIsSuccess())
+            {
+                _lastErrorContent = _client.ReadAsString();
+            }
+        }
+
+        [When(@"I wait for the print process to complete")]
+        public async Task WhenIWaitForThePrintProcessToComplete()
+        {
+            // Wait for bus/outbox to be idle
+            await _testHost.ThenIWaitBackgroundBusToIdleAndOutboxToBeEmpty().ConfigureAwait(false);
```

Comment:
move this in the feature file as Verb 

And I Wait background bus to idle and ...